cpp/ql/lib/semmle/code/cpp/ir/dataflow/MustFlow.qll

```diff
@@ -48,18 +53,21 @@ abstract class MustFlowConfiguration extends string {
    */
   final predicate hasFlowPath(MustFlowPathNode source, MustFlowPathSink sink) {
     this.isSource(source.getInstruction()) and
-    source.getASuccessor+() = sink
+    source.getASuccessor*() = sink
```

Comment:
Any idea why this was `+()` in the first place?

Reply:
Likely because it wasn't possible for the number of steps to be 0 steps in the queries that used the library before. Or I simply mistyped it when I added the library 😂

Reply:
OK, as long as we're not likely to be accidentally causing false positives to appear elsewhere.

All of our other dataflow-type libraries work this way, so it seems more likely to be a good thing.

Reply:
> OK, as long as we're not likely to be accidentally causing false positives to appear elsewhere.

Indeed. DCA didn't reveal anything in particular, and I can't imagine 0 steps would be a problem for any of them.